src/test/rpc/RPCCall_test.cpp

```diff
@@ -77,6 +79,7 @@ static RPCCallTestData const rpcCallTestArray [] =
     "method" : "account_channels",
     "params" : [
       {
+         "api_version" : 1,
```

Comment:
if I understand correctly, this version text/field is always replaced with the max supported (?). If that's the case, I would prefer just a token string here (e.g. `%MAX_API_VER%`....) and then you search/replace that token in the update method (it gives a better sense of what's actually happening). If I've misunderstood what's going on, please ignore.

Reply:
Great, thanks! I changed the code.